demo/sfml_opengl3/nuklear_sfml_gl3.h

```diff
@@ -442,7 +442,9 @@ nk_sfml_handle_event(sf::Event* evt)
         } else nk_input_motion(ctx, evt->touch.x, evt->touch.y);
         return 1;
     } else if(evt->type == sf::Event::TextEntered) {
-        nk_input_unicode(ctx, evt->text.unicode);
+		if (evt->text.unicode != 8) {
```

Comment:
Please add explanation comment what this check means and then I'll merge this :wink:. Thanks!